### PR TITLE
Remove autofocus on twitter share control button

### PR DESCRIPTION
**Pull request by MarianneLa**

### CODE REVIEW

File: src/modules/Map/Map/components/ShareControl/ShareControl.js [hunk 1]
```diff
@@ -152,10 +152,9 @@ export class ShareControl extends AbstractMapControl {
               content={translate('terralego.map.share_control.share', {
                 context: network.charAt(0).toUpperCase() + network.slice(1),
               })}
+              openOnTargetFocus={false}
             >
-              <Button
-                onClick={this.share(network)}
```
Comment (hadrienl) [inline src/modules/Map/Map/components/ShareControl/ShareControl.js]: Why did you remove the onClick ?

Comment (MarianneLa) [reply src/modules/Map/Map/components/ShareControl/ShareControl.js]: mistake...
